particles/Tutorial/Javascript/Demo/TTTGame.arcs

```diff
@@ -4,133 +4,134 @@ import 'TTTGameSchemas.arcs'
 import 'TTTGameData.arcs'
 
 particle Game in './source/TTTGame.js'
-  inout Person {name, avatar, id} playerOne
-  inout Person {name, avatar, id} playerTwo
-  inout GameState gameState
-  inout Move {move} playerOneMove
-  inout Move {move} playerTwoMove
-  inout [Event] events
-  consume root
-    provide gameSlot
+  playerOne: reads writes Person
+  playerTwo: reads writes Person
+  gameState: reads writes GameState
+  playerOneMove: reads writes Move
+  playerTwoMove: reads writes Move
+  events: reads writes [Event]
+  root: consumes
+    gameSlot: provides
 
 particle TicTacToeBoard in './source/TTTBoard.js'
-  in GameState {board, currentPlayer} gameState
-  out [Event] events
-  consume boardSlot
+  gameState: reads GameState
+  events: writes [Event]
+  boardSlot: consumes
 
 particle HumanPlayer in './source/TTTHumanPlayer.js'
-  in GameState {currentPlayer} gameState
-  in [Event] events
-  in Person {id} player
-  out Move {move} myMove
+  gameState: reads GameState
+  events: reads [Event]
+  player: reads Person
+  myMove: writes Move
 
 particle RandomComputer in './source/TTTRandomComputer.js'
-  out Move {move} computerMove
-  in GameState {board, currentPlayer} gameState
-  in Person {id} player
+  computerMove: writes Move
+  gameState: reads GameState
+  player: reads Person
```

Comment:
are you not specifying the inline schema properties here on purpose - existing `Person {id}` vs new `Person`?